interface/billing/sl_eob_invoice.php

```diff
@@ -23,16 +23,17 @@
  * @author  Terry Hill <terry@lillysystems.com>
  * @link    http://www.open-emr.org
```

Comment:
The following prior code changes need to be added:
https://github.com/zbig01/openemr/commit/284dfc3df7cb8f49168a140456bde2b3fdb9d1d3#diff-daddf8c35166ff00e7c044ee888b2fdb

Otherwise, this script is good to go (and history has been checked)

Reply:
Done